proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,175 @@
+# Instructor Training Selection
+
+**Warning: This is only a draft.**
+
+**Warning: This only apply for the two-day training course in December 2015.**
+
+## Terminology
+
+-   An **instructor** is someone who teaches a workshop. Instructor training (IT)
+    prepares them to do this.
+
+## Why Select Attends for Instructor Training
+
+SCF/DC wants to train as more instructors as possible
+but demand for instructor training is higher than what SCF/DC could offer
+at the moment.
+For the free/open online training SCF/DC hopes to satisfy
+the demand for instructor training from their community
+that isn't associate with one partner or affiliate
+and could participate at the Instructor Training
+delivered for partners and affiliates.
+Therefore if you work for one of the SCF/DC Partnering Sites,
+please inquire first at your organisation if you could undergo IT hosted by them.
+The Instructor Training described in this document is addressed
+at people who do not have such affiliation
+and hence have practically no alternative for getting trained as an instructor.
+
+## Eligibility
+
+### Groups
+
+-   With 4-6 people.
+-   Co-locate in one room for the two days
+-   Commit to jointly running one workshop no later than April 1st, 2016.
+
+### Members
+
+-   Be on the current waiting list
+    or
+    demonstrate sufficient competence with core tools used in the lessons they will teach.
+
+-   Watch a short video of someone teaching part of our core lessons and
+    write a summary of what was taught and how. Participants in the Fall
+    2015 pilot will be required to complete this step, and we will
+    select segments from the videos from SciPy 2015.
+
+## Application
+
+Groups **must** send a cover letter to [ADDRESS TO SEND COVER LETTER]()
```

Comment:
I have set up 'training-help@software-carpentry.org' to forward to 
myself and Karin Lagesen - shall we use that instead?